### PR TITLE
[BAEL-4532]a.getClass() vs A.class

### PR DESCRIPTION
**Pull request by sk1418**

### CODE REVIEW

File: core-java-modules/core-java-lang-3/src/main/java/com/baeldung/getclassobject/Animal.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.baeldung.getclassobject;
+
+public class Animal {
+   protected int numberOfEyes;
```
Comment (eric-martin) [inline core-java-modules/core-java-lang-3/src/main/java/com/baeldung/getclassobject/Animal.java:4]: 4 space indent (I've fixed the article)

Comment (sk1418) [reply core-java-modules/core-java-lang-3/src/main/java/com/baeldung/getclassobject/Animal.java]: Thanks.
Fixed.

File: core-java-modules/core-java-lang-3/pom.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project
-    xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+        xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (KevinGilmore) [inline core-java-modules/core-java-lang-3/pom.xml:3]: As a general guideline, we shouldn't make changes that are whitespace-only.

**Review by KevinGilmore — COMMENTED** [review]
Looks like the changes to pom.xml are only changes to the whitespace.

File: core-java-modules/core-java-lang-3/pom.xml (hunk 2 above)
Comment (sk1418) [reply core-java-modules/core-java-lang-3/pom.xml:3]: @KevinGilmore Thanks. I've reverted the changes to this file.

**Pull request merged**